### DISCUSSION TITLE
Helm chart: no option to configure storage capacity

### DISCUSSION THREAD

**Discussion by romalukin** [aquasecurity/trivy-operator#1225] [Help & Support]
At first I'd like to say thanks for your amazing product. 
In helm chart (trivy-server part), we can't specify the size value of PVC, it is hardcoded to 5Gi.

https://github.com/aquasecurity/trivy-operator/blob/6b29e21f1ce8f34afc22ca39d528f644ba92d205/deploy/helm/templates/trivy-server.yaml#L46-L51

I'm experimenting now, and the server has already taken up all the space. Maybe it will be better to make it possible to specify the amount of resources in values file?

**Comment by maltemorgenstern** [comment]
Hey @romalukin,
#1801 added support for configuring the PVC size:
```
trivy:
  storageSize: "10Gi"
```

This is available since v0.18.3: https://github.com/aquasecurity/trivy-operator/releases/tag/v0.18.3
